thunder/dynamo/report.py

```diff
@@ -1058,11 +1090,13 @@ def get_thunder_fxgraph_reports(
         check_torch_compile_runnability(fn, *args, **kwargs)
 
     reports = fx_report(fn, *args, **kwargs)
+    print(reports)
```

Comment:
In the future we'll probably want to replace these prints with a logger that can be silenced or easily redirected